backend/dataall/api/Objects/DataPipeline/schema.py

```diff
@@ -1,6 +1,8 @@
 from ... import gql
 from .resolvers import *
 from ...constants import DataPipelineRole
+from dataall.api.Objects.Environment.resolvers import resolve_environment
```

Comment:
[Clarification ] Nice addition - Instead of using a resolver in each object that basically did the same (retrieve environment), we use the resolve_environment generic method to avoid duplications